libr/core/cmd_anal.c

```diff
@@ -4158,6 +4211,17 @@ static void cmd_anal_esil(RCore *core, const char *input) {
 		}
 		break;
 	case 'x': { // "aex"
+		if (input[1] == 'n'){
+			ut64 addr = r_num_math (core->num, input + 3);
```

Comment:
i would not trust input + 3 is before the end of the string, you are only checking input[0] and input[1], so input[2] canbe null and input+3 will be an oob read